doc/source/whatsnew/v0.16.0.txt

```diff
@@ -117,6 +117,20 @@ Enhancements
 - Added ``StringMethods.ljust()`` and ``rjust()`` which behave as the same as standard ``str`` (:issue:`9352`)
 - ``StringMethods.pad()`` and ``center()`` now accept `fillchar` option to specify filling character (:issue:`9352`)
 
+
+
+- ``Series`` now supports bitwise operation for integral types (:issue:`9016`)
+  .. code-block:: python
+  In [2]: pd.Series([0,1,2,3], list('abcd')) | pd.Series([4,4,4,4], list('abcd'))
```

Comment:
indent 2 spaces here (the code under the directive), otherwise won't render correctly